Client/Frontend/Library/BookmarkDetailPanel.swift

```diff
@@ -121,6 +121,7 @@ class BookmarkDetailPanel: SiteTableViewController {
                 self.navigationController?.popViewController(animated: true)
             }
         }
+        updateSaveButton(text: bookmarkItemURL)
```

Comment:
Prior to doing validation here, check if we're creating a new bookmark and if so, pre-set the `URL` field to contain the string `https://`.

```
if isNew, bookmarkNodeType == .bookmark {
    bookmarkItemURL = "https://"
}
```